Makefile.prow

```diff
@@ -0,0 +1,11 @@
+# Copyright Contributors to the Open Cluster Management project
+
+-include /opt/build-harness/Makefile.prow
+
+.PHONY: bundle-latest-image
+bundle-latest-image:
+	bash ./scripts/bundle-image-pickup.sh -s
+
+.PHONY: e2e
+e2e:
+	echo "Running e2e test..."
```

Comment:
Update message for transparency and help us debug later.
```suggestion
	echo "[TODO] Run e2e test here."
```

Reply:
Added suggestion to `Makefile`.